### PR TITLE
Refactor build sh

### PR DESCRIPTION
**Pull request by ocefpaf**
Continue #97

### CODE REVIEW

File: recipe/build.sh [hunk 1]
```diff
@@ -2,27 +2,23 @@
...
-        --with-tcltk-libs="-L$PREFIX/lib -ltcl8.5 -ltk8.5" \
-        CPPFLAGS="-I$PREFIX/include" \
-        LDFLAGS="-L$PREFIX/lib -Wl,-rpath=$PREFIX/lib,--no-as-needed"
+if [ $(uname) == Darwin ]; then
+  export CFLAGS="-I$PREFIX/include $CFLAGS"
+  export LDFLAGS="-Wl,-rpath,$PREFIX/lib -L$PREFIX/lib -headerpad_max_install_names $LDFLAGS"
+  sed -i -e "s/@OSX_ARCH@/$ARCH/g" Lib/distutils/unixccompiler.py
+elif [ $(uname) == Linux ]; then
+  export CPPFLAGS="-I$PREFIX/include"
+  export LDFLAGS="-L$PREFIX/lib -Wl,-rpath=$PREFIX/lib,--no-as-needed"
```
Comment (ocefpaf) [inline recipe/build.sh]: Not sure if this is OK or if these must be in the `configure` line.

Comment (xmnlab) [inline recipe/build.sh]: 👍

**Comment by ocefpaf** [comment]
Rebased for latest `master` (Python 3.6). If merged we need to port this to 3.5 too.

**Comment by jjhelmus** [comment]
LGTM.  Could you change the PR title to include a mention of enabling loading of sqlite extensions.  This should be mentioned in the git history in some manner.

**Comment by ocefpaf** [comment]
It was. I guess that my rebasing removed it. Hold on...

**Comment by ocefpaf** [comment]
Better?

**Comment by jjhelmus** [comment]
> Better?

Great, will merge once the CI are green.  Thanks.

**Comment by ocefpaf** [comment]
> Great, will merge once the CIs are green. Thanks.

No problem. Should we backport this o Python 3.5?

**Comment by jjhelmus** [comment]
> Should we backport this o Python 3.5?

I think so, it's a good addition.

**Comment by jjhelmus** [comment]
Thanks @ocefpaf

**Pull request merged**